projects/project-player-library/src/lib/shared/add-link-popup/add-link-popup.component.css

```diff
@@ -0,0 +1,3 @@
+.mat-dialog-container {
```

Comment:
Is this styling required and where is this class used in html?

Reply:
it was for angular material dialog popup, i don't think it is required and removed it